behat.yml.dist

```diff
@@ -7,6 +7,7 @@ default:
         - Drupal\DrupalExtension\Context\DrupalContext
         - Drupal\DrupalExtension\Context\MessageContext
         - Drupal\DrupalExtension\Context\MinkContext
+        - Drupal\Tests\oe_content\Behat\FeatureContext
```

Comment:
FeatureContext normally we use for component-specific tests and normally not shareable with other components. But this approach was important for EWCMS. So for component looks like it is fine to do this (if @ademarco agree).

Reply:
Which steps are required from here? @22Alexandra

Reply:
Filling in Timeline field values.